tests/scripts/pulp_rpm/test_rpm_sync_publish.sh

```diff
@@ -42,6 +42,14 @@ expect_succ pulp rpm repository update --repository "cli_test_rpm_repository" --
 expect_succ pulp rpm repository show --repository "cli_test_rpm_repository"
 test "$(echo "$OUTPUT" | jq -r '.description')" = "null"
 
+# sqlite metadata removal from pulp-cli (pulp/pulp_rpm#3328)
+if pulp debug has-plugin --name "rpm" --specifier ">3.24.0"
```

Comment:
```suggestion
if pulp debug has-plugin --name "rpm" --specifier ">=3.24.0"
```
I cannot imagine it will be removed in a z stream...

Reply:
Sorry, forget to talk about that. I still need to check when it will be released with Daniel and Ina, I was using it as a placeholder.

Reply:
@dralley and @ipanova , wdyt?

Reply:
And 3.25 is certain? Is it merged in pulp_rpm already?

Reply:
It's not merged but it will be shortly